python/paddle/fluid/tests/unittests/mkldnn/test_conv2d_int8_mkldnn_op.py

```diff
@@ -23,13 +23,12 @@
 
 
 def conv2d_forward_refer(input, filter, group, conv_param):
-    out, in_n, out_h, out_w, out_c = conv2d_forward_naive(input, filter, group,
-                                                          conv_param)
+    out, _, _, _, _ = conv2d_forward_naive(input, filter, group, conv_param)
     return out
 
 
-@unittest.skipIf(not core.supports_bfloat16(),
-                 "place does not support BF16 evaluation")
+@unittest.skipIf(not core.supports_int8(),
```

Comment:
If we don't want to be stopped by CI-Approval, you can use OpTestTool to skip these tests if int8 is not supported by place. The behavior is identical to using pure "@unittest.skipIf", but since it is a wrapper regex cannot catch that and CI-Approval does not complain about that
`
@OpTestTool.skip_if(not core.supports_int8(), "place does not support int8 computation")
`